ui/src/containers/BuildConfigDialog.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import CreateBuildConfigDialog from '../components/build/CreateBuildConfigDialog';
+import { createBuildConfig } from '../actions/buildConfigs';
+
+class BuildConfigDialog extends Component {
+  buildUpdated = configUpdate => {
+    const config = Object.assign({}, configUpdate.config);
+    this.props.createBuildConfig(config);
```

Comment:
BuildConfig update is not yet implemented.

Reply:
renamed